asciidoctorj-core/build.gradle

```diff
@@ -32,3 +34,19 @@ jrubyPrepareGems << {
 //jruby {
 //	execVersion = '1.7.20'
 //}
+
+jar {
+  manifest {
+    symbolicName = 'org.asciidoctor'
+    instruction 'Export-Package',
+      "org.asciidoctor;version=\"${version}\"",
+      "org.asciidoctor.ast;version=\"${version}\"",
+      "org.asciidoctor.converter;version=\"${version}\"",
+      "org.asciidoctor.converter.spi;version=\"${version}\"",
+      "org.asciidoctor.extension;version=\"${version}\"",
+      "org.asciidoctor.extension.spi;version=\"${version}\""
+    instruction 'Import-Package',
+      'com.beust.jcommander;resolution:=optional',
```

Comment:
Do you think this line is necessary? Would the CLI ever be used from an OSGi environment. It's not really intended to be an API atm. But perhaps I'm missing a use case.